### PR TITLE
PAAS-2145 fix redeploy/upgrade breaking APM, set apm in setenv before restart karaf on install

### PR DESCRIPTION
**Pull request by laurentfufu**
JIRA issue: https://jira.jahia.org/browse/PAAS-2145

Short description:

### CODE REVIEW

**Review by kevinvandel — CHANGES_REQUESTED** [review]
We need to add an action to the migration package to reinstall the APM when it's missing

**Comment by laurentfufu** [comment]
a fixAPM action have been added to migration v15 manifest

File: packages/jcustomer/migrations/migrate-to-v15.yaml [hunk 1]
```diff
@@ -37,3 +45,72 @@ actions:
...
+        }
+        """
+
+    - script: |-
+        const rep = ${response.out.toJSON()}
+        const result = {"result": 0}
+        return Object.assign({}, result, rep)
+
+    - set:
+        apm_loaded: ${response.apm_loaded}
```
Comment (Ronan-Gorain) [inline packages/jcustomer/migrations/migrate-to-v15.yaml:92]: This variables seems to be unused

Comment (laurentfufu) [reply packages/jcustomer/migrations/migrate-to-v15.yaml]: it is now 6 feet under

**Review by Ronan-Gorain — APPROVED** [review]

**Pull request merged**